test/e2e/app-dir/crypto-globally-available/app/handler/route.ts

```diff
@@ -0,0 +1,9 @@
+export function GET() {
+  return new Response(
+    typeof globalThis.crypto === 'object'
+      ? 'crypto is available'
+      : 'crypto is not available'
+  )
+}
+
+export const runtime = 'nodejs'
```

Comment:
Set explicitly so the test will work indefinitely in the future. Note: `"edge"` already exposes `globalThis.crypto` for us.